src/components/BareMetalHosts/tests/BaremetalHostStatus.test.js

```diff
@@ -17,4 +20,16 @@ describe('<BaremetalHostStatus />', () => {
     const component = shallow(testBaremetalHostStatusSuccess());
     expect(component).toMatchSnapshot();
   });
+  it('renders a match profile state', () => {
```

Comment:
it is expanding.. Can you please match the snapshots in a for loop. You can look at VmStatus.test.js for inspiration.

Reply:
Thanks for the hint. Done